Makefile

```diff
@@ -211,6 +223,14 @@ check: utility mods
 	@echo "Checking for code style violations in OpenRA.Test..."
 	@mono --debug OpenRA.Utility.exe ra --check-code-style OpenRA.Test
 
+NUNIT_CONSOLE := $(shell test -f nunit-console.exe && echo mono nunit-console.exe || which nunit2-console 2>/dev/null || which nunit3-console 2>/dev/null || which nunit-console 2>/dev/null)
+nunit: test_dll
+	@echo
+	@echo "Checking unit tests..."
+	@if [[ "$(NUNIT_CONSOLE)" = "" ]] ;then echo 'nunit[2|3]-console not found!'; echo 'Was "make dependencies" called or is NUnit installed?'>&2; exit 1; fi
```

Comment:
The error message should also point the user to `make help`. It wasn't obvious to me why `make nunit` still failed even though I had run `make dependencies`.